mod/developers/views/default/admin/develop_tools/entity_explorer.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+echo '<p>' . elgg_echo('developers:entity_explorer:help') . '</p>';
+
+echo elgg_view_form('developers/entity_explorer', [
+	'action' => 'admin/develop_tools/entity_explorer',
+	'method' => 'GET',
+	'disable_security' => true,
+]);
+
+$guid = get_input('guid');
+if ($guid === null) {
+	return;
+}
+
+$show_hidden = access_show_hidden_entities(true);
+$entity = get_entity($guid);
+if (!$entity) {
+	echo elgg_echo('notfound');
+	return;
+}
+
+// Entity Information
+$entity_rows = ['type', 'subtype', 'owner_guid', 'site_guid', 'container_guid', 'access_id', 'time_created', 'time_updated', 'last_action', 'enabled'];
+
+if ($entity instanceof ElggUser) {
+	$entity_rows = array_merge($entity_rows, ['name', 'username', 'email', 'language', 'banned', 'admin', 'last_action', 'prev_last_action', 'last_login', 'prev_last_login']);	
+}
+
+if ($entity instanceof ElggGroup) {
+	$entity_rows = array_merge($entity_rows, ['name', 'description']);	
+}
+
+if ($entity instanceof ElggSite) {
+	$entity_rows = array_merge($entity_rows, ['name', 'description', 'url']);	
+}
+
+$entity_info = '<table class="elgg-table">';
+
+foreach ($entity_rows as $entity_row) {
+	$entity_info .= '<tr>';
+	$entity_info .= '<td>' . $entity_row . '</td><td>' . $entity->$entity_row . '</td>';
+	$entity_info .= '</tr>';
+}
+
+$entity_info .= '</table>';
+	
+echo elgg_view_module('inline', elgg_echo('developers:entity_explorer:info:attributes'), $entity_info);
+
+// Metadata Information
+$entity_metadata = elgg_get_metadata(['guid' => $guid, 'limit' => false]);
```

Comment:
We should probably use `elgg_list_*` where possible with proper offset keys, in case these get too much on a large site.